common/source/docs/common-matekh743-wing.rst

```diff
@@ -0,0 +1,123 @@
+.. _common-matekh743-wing:
+
+==================
+Mateksys F765-Wing
+==================
+
+.. image:: ../../../images/matekh743-wing.png
+     :target: ../_images/matekh743-wing.png
+    
+
+the above image and some content courtesy of `mateksys.com <http://www.mateksys.com/>`__
+
+.. note::
+
+   Support for this board is available with ArduPilot Plane 4.0.6, Copter 4.0.4, and higher
+
+Specifications
+==============
+
+-  **Processor**
+
+   -  STM32H743VIT6  ARM (216MHz)
+
+
+-  **Sensors**
+
+   -  InvenSense MPU6000 IMU (accel, gyro) & ICM20602
+   -  MS5611 barometer
+   -  Voltage & 132A current sensor
+
+
+-  **Power**
+
+   -  9V ~ 36V DC input power
+   -  5V 2A BEC for peripherals
+   -  9/12V 2A BEC for video
+   -  5/6/7.2V 8A BEC for servos
+
+
+-  **Interfaces**
+
+   -  7x UARTS
+   -  13x PWM outputs
+   -  1x RC input PWM/PPM, SBUS
+   -  2x I2C ports for external compass, airspeed sensor, etc.
+   -  SPI4 port
+   -  USB port (with remote cabling)
+   -  CAN port
+   -  6 ADC
+   -  Buzzer and Safety Switch
+   -  Dual Switchable Camera inputs
+   -  Built-in OSD
+   -  microSD card
+   -  Second battery monitor input pins
+
+
+-  **Size and Dimensions**
+
+   - tbd mm x tbd mm x tbd mm
+   - tbd g
+
+
+   
+Default UART order
+==================
+
+- SERIAL0 = console = USB
+- SERIAL1 = Telemetry1 = UART7 (support CTS and RTS signaling)
+- SERIAL2 = Telemetry2 = USART1
+- SERIAL3 = GPS1 = USART2
+- SERIAL4 = GPS2 = USART3
+- SERIAL5 = USER = UART8
+- SERIAL6 = USER = UART4
+- SERIAL7 = USER = UART6 (TX only unless :ref:`BRD_ALT_CONFIG<BRD_ALT_CONFIG>` = 1, then RX available also)
+
+RC input is configured on the R6 (UART6_RX) pin. It supports all RC protocols, however for FPort  :ref:`BRD_ALT_CONFIG<BRD_ALT_CONFIG>` should be set to 1 and configured as described in :ref:`FPort<common-FPort-receivers>` section.
+
+Serial port protocols (Telem, GPS, etc.) can be adjusted to personal preferences.
+
+Dshot capability
+================
+
+All motor/servo outputs are Dshot and PWM capable. However, mixing Dshot and normal PWM operation for outputs is restricted into groups, ie. enabling Dshot for an output in a group requires that ALL outputs in that group be configured and used as Dshot, rather than PWM outputs. The output groups that must be the same (PWM rate or Dshot, when configured as a normal servo/motor output) are: 1/2, 3/4/5/6,  or 7/8/9/10. Outputs 11,12, and 13 are not Dshot capable.
```

Comment:
PWM13(LED2812) TIM1_CH1 is Dshot compable,   H7 TIM15_CH1/CH2 (PWM11/12)should work with Dshot also.

Reply:
done